### PR TITLE
Add hints to the DAML Intro docs to encourage downloading the code.

### PR DESCRIPTION
**Pull request by bame-da**
CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Comment by axos88-da** [comment]
@bame-da Thanks for reacting so quickly to my feedback, but I personally would rather that the docs could be followed step-by-step from the text rather than having to download the complete code. I prefer to build the code as I go through the explanation and the tutorial, as sometimes it is harder to follow how the resulting code came to be.

Adding a simple note about the missing import along with the note added in this PR would allow to go either way.

File: docs/source/daml/intro/1_Token.rst [hunk 1]
```diff
@@ -12,6 +12,10 @@ To begin with, you're going to write a very small DAML template, which represent
 - Contracts
 - Signatories
 
+.. hint::
+
+  Remember that you can download all the code for this section by running ``daml new 1_Token daml-intro-1``
```
Comment (cocreature) [inline docs/source/daml/intro/1_Token.rst:17]: That isn’t downloading anything. Maybe `Remember that you can checkout all the code for this section locally by running …`?

File: docs/source/daml/intro/5_Restrictions.rst [hunk 2]
```diff
@@ -33,6 +37,9 @@ The ``ensure`` keyword takes a single expression of type ``Bool``. If you want t
   :start-after: ensure
   :end-before: -- RESTRICTED_IOU_END
 
+.. hint::
```
Comment (bame-da) [inline docs/source/daml/intro/5_Restrictions.rst:40]: @axos88-da I added this specific hint at the highlighted section as well.

File: docs/source/daml/intro/1_Token.rst (hunk 1 above)
Comment (bame-da) [reply docs/source/daml/intro/1_Token.rst]: went with "load"

**Comment by cocreature** [comment]
This has conflicts now that need to be resolved.

**Comment by cocreature** [comment]
@bame-da if you still want to merge this it needs rebasing.

**Comment by bame-da** [comment]
lost track of this PR, thanks.

**Pull request merged**